tests/regression/71-strings/01-string_literals.c

```diff
@@ -0,0 +1,84 @@
+// PARAM: --disable ana.base.limit-string-addresses --enable ana.int.interval
+
+#include <goblint.h>
+#include <string.h>
+#include <stdlib.h>
+
+char* hello_world() {
+    return "Hello world!";
+}
+
+void id(char* s) {
+    s = s;
+}
```

Comment:
I'm not sure what this function is trying to achieve. The identity copy of the pointer/string literal is only in the scope of this function, so even something like `s = s + 1` wouldn't influence the passed string in the caller.

Reply:
Yes, I messed that one up. Thanks for pointing it out!